cfme/common/provider.py

```diff
@@ -50,6 +52,7 @@ class BaseProvider(Taggable, Updateable, SummaryMixin):
     _properties_region = None
     add_provider_button = None
     save_button = None
+    appliance = get_or_create_current_appliance()
```

Comment:
Right and that's exactly what it does. Returns a one time, what _was_ the current_appliance.